CONTRIBUTING.md

```diff
@@ -2,33 +2,22 @@
 
 Contributions are highly welcome, however, except for very small changes, kindly file an issue and let's have a discussion before you open a pull request.
 
-## Building The Project
+## Building the Project
 
 Clone this repo:
 
-```bash
-git clone https://github.com/tonerdo/coverlet
-```
+    git clone https://github.com/tonerdo/coverlet.git
+    cd coverlet
 
-Change directory to repo root:
+Building, testing, and packing use all the standard dotnet commands:
 
-```bash
-cd coverlet
-```
+    dotnet build
+    dotnet test
```

Comment:
```suggestion
    dotnet test /p:CollectCoverage=true /p:CoverletOutputFormat=opencover /p:Include=[coverlet.*]*"
```

Reply:
We should exlude from .gitignore all generated file
![image](https://user-images.githubusercontent.com/7894084/58547419-994f3e00-8207-11e9-9a97-ba22bf728bd1.png)

Reply:
@AArnott if check resolved but I think we need to update .gitignore before, something like 
`/test/**/coverage.json`
`/test/**/coverage.opencover.xml`

Reply:
Done. Was this a new issue introduced by this PR?

Reply:
yep, with build.csproj we use `OutputPath` https://github.com/tonerdo/coverlet/blob/11c15b59eba109f2333254773f82bb3137214e22/build.proj

Reply:
Uhm...look a little closer it's not clear at the moment why this issue come out now :expressionless:

Reply:
Alright well, it is a good change anyway.

Reply:
A lot of changes on build in these days surely I missing the reason inside commits, BTW it a good change to me too.